python/lsst/cell_coadds/_multiple_cell_coadd.py

```diff
@@ -200,3 +206,132 @@ def explode(self, pad_psfs_with: float | None = None) -> ExplodedCoadd:
             Exploded version of the coadd.
         """
         return ExplodedCoadd(self, pad_psfs_with=pad_psfs_with)
+
+    def writeFits(self, *args, **kwargs):
+        """Write this coadd to one or more FITS files."""
```

Comment:
Type checkers are unhappy if I don't define `writeFits` here but use it elsewhere. This is purely to please them but other ideas would be nice.